index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html>
+<head>
+<!-- 1. Title - make it meaningful. -->
+	<title>Bryan Miller</title>
+	<link href="https://fonts.googleapis.com/css?family=Anton|Sansita" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="/css/styles.css">
+</head>
+<body>
+<header>
+<!-- 2. Header element `<h1>` - This is the main headline; include your name and what this page is. -->
+	<h1>Bryan Miller</h1>
+	<h3>this page is about web design.</h3>
+</header>
+<!-- 3. Photo -->
+	<img class ="profile-pic" src="/images/bryan_profile_pic.jpg" alt="profile picture">
+<!-- 4. Article element containing your bio -->
+	<article>
+		<h2>Biographyish</h2>
+		<table>
+			<tr>
+    			<th>Name:</th>
+    			<td>Bryan</td>
+    			<td>Miller</td>
+  			</tr>
+  			<tr>
+    			<th>Born:</th>
+    			<td>January</td>
+  			  	<td>1971</td>
+ 			</tr>
+  			<tr>
+    			<th>Place:</th>
+    			<td>Dallas</td>
+  			  	<td>Texas</td>
+ 			</tr>
+		</table>
+		<p>I was born in Texas, and grew up with the best BBQ in the world.  Totally irrelavant to the conversation at hand, but I seem to include it every time I speak of Texas.  When it comes to design, good design is always instantly recognizable, and so is bad design.  Working in the sign industry, we got to do some really cool things, and we made our share of bad ideas come to life, because the customer was paying for it.  It made me appreciate clean simple design all the more.</p>
```

Comment:
This is a good start but it says nothing about the fact that you are trying to become a software developer.